services/chatbot/app/server.py

```diff
@@ -95,10 +96,22 @@ def _extract_thread_id(
     config: Dict[str, Any],
     request: Request
 ) -> Dict[str, Any]:
-    # TODO: Validate JWT token
-    # TODO: Extract conversation id from the JWT token
     configurable = config.get("configurable", {})
-    configurable["thread_id"] = "1"
+    jwt_bearer_token = request.headers.get("Authorization", None)
+    assert(jwt_bearer_token is not None, "Authorization header must be set")
+    assert(jwt_bearer_token.startswith("Bearer "), "Sanity check")
+    jwt_token = jwt_bearer_token.replace("Bearer ", "", 1)
+
+    claims = jwt.decode(
+        jwt_token,
+        options = {
+            # TODO: Validate JWT token
+            "verify_signature": False
```

Comment:
JWT verification is temporary disabled due to issues with the different cryptographic elliptic curve primitives in .Net and Python libraries.